templates/shortcode-receipt.php

```diff
@@ -146,87 +142,14 @@
-				<td><?php echo date_i18n( give_date_format(), strtotime( $meta['date'] ) ); ?></td>
-			</tr>
-		<?php endif; ?>
-
-		<?php if ( filter_var( $give_receipt_args['price'], FILTER_VALIDATE_BOOLEAN ) ) : ?>
-			<tr>
-				<td scope="row"><strong><?php esc_html_e( 'Total Donation:', 'give' ); ?></strong></td>
-				<td><?php echo give_payment_amount( $payment->ID ); ?></td>
-			</tr>
-		<?php endif; ?>
-
-		<tr>
-			<td scope="row" class="give_receipt_payment_status">
-				<strong><?php esc_html_e( 'Donation:', 'give' ); ?></strong></td>
-			<td class="give_receipt_payment_status"><?php echo $donation; ?></td>
-		</tr>
-
-		<?php if ( filter_var( $give_receipt_args['payment_status'], FILTER_VALIDATE_BOOLEAN ) ) : ?>
-			<tr>
-				<td scope="row" class="give_receipt_payment_status">
-					<strong><?php esc_html_e( 'Donation Status:', 'give' ); ?></strong></td>
-				<td class="give_receipt_payment_status <?php echo esc_attr( $status ); ?>"><?php echo esc_html( $status_label ); ?></td>
-			</tr>
-		<?php endif; ?>
-
-		<?php if ( filter_var( $give_receipt_args['payment_id'], FILTER_VALIDATE_BOOLEAN ) ) : ?>
-			<tr>
-				<td scope="row"><strong><?php esc_html_e( 'Donation ID:', 'give' ); ?></strong></td>
-				<td><?php echo give_get_payment_number( $payment->ID ); ?></td>
-			</tr>
-		<?php else : ?>
-			<tr>
-				<td scope="row"><strong><?php esc_html_e( 'Payment:', 'give' ); ?></strong></td>
-				<td><?php esc_html_e( 'Details:', 'give' ); ?></td>
-			</tr>
-		<?php endif; ?>
-
-		<?php if ( filter_var( $give_receipt_args['payment_key'], FILTER_VALIDATE_BOOLEAN ) ) : ?>
-			<tr>
-				<td scope="row"><strong><?php esc_html_e( 'Payment Key:', 'give' ); ?></strong></td>
-				<td><?php echo give_get_meta( $payment->ID, '_give_payment_purchase_key', true ); ?></td>
-			</tr>
-		<?php endif; ?>
-
-		<?php if ( filter_var( $give_receipt_args['payment_method'], FILTER_VALIDATE_BOOLEAN ) ) : ?>
-			<tr>
-				<td scope="row"><strong><?php esc_html_e( 'Payment Method:', 'give' ); ?></strong></td>
-				<td><?php echo give_get_gateway_checkout_label( give_get_payment_gateway( $payment->ID ) ); ?></td>
-			</tr>
-		<?php endif; ?>
-
-		<?php
-		//No fees built in just yet...
-		//@TODO: Fees
-		if ( ( $fees = give_get_payment_fees( $payment->ID, 'fee' ) ) ) : ?>
-			<tr>
-				<td scope="row"><strong><?php esc_html_e( 'Fees:', 'give' ); ?></strong></td>
-				<td>
-					<ul class="give_receipt_fees">
-						<?php foreach ( $fees as $fee ) : ?>
-							<li>
-								<span class="give_fee_label"><?php echo esc_html( $fee['label'] ); ?></span>
-								<span class="give_fee_sep">&nbsp;&ndash;&nbsp;</span>
-								<span
-									class="give_fee_amount"><?php echo give_currency_filter( give_format_amount( $fee['amount'] ) ); ?></span>
-							</li>
-						<?php endforeach; ?>
-					</ul>
-				</td>
-			</tr>
-		<?php endif; ?>
+		<?php foreach( $give_receipt_args['donation_receipt'] AS $receipt_item ){ ?>
```

Comment:
@ravinderk I usually keep `as` capital because it keeps me differentiated with variables before and after. I can change that, if we want to make it common throughout the Give Core